src/utils/DateUtils.ts

```diff
@@ -0,0 +1,16 @@
+class DateUtils {
+	static getDaysBetweenDates(date1: Date, date2: Date): number {
+		const diff = date1.getTime() - date2.getTime();
+
+		return Math.floor(diff / (1000 * 3600 * 24));
+	}
+
+	static formatDaysAgo(days: number): string {
+		if (days < 0) throw new Error("Number has to be positive");
+		if (days === 0) return "Today";
+		if (days === 1) return "Yesterday";
```

Comment:
Can these be in lowercase, please? I feel as if a util shouldn't have any capitalisation.

Reply:
My thinking was if it was needed at the start of a sentence it could be used like this, in other cases it would be easy to add `.toLowerCase()`.

Reply:
I guess, but it's sort of assuming use-case when utils are meant to be standalone.